src/routes/messages/entities/message.entity.ts

```diff
@@ -16,7 +68,9 @@ export class Message {
   logoUri: string | null;
   @ApiPropertyOptional({ type: String, nullable: true })
   name: string | null;
-  @ApiProperty()
+  @ApiProperty({
+    oneOf: [{ type: 'string' }, { $ref: getSchemaPath(EIP712TypedData) }],
+  })
```

Comment:
I can't comment on the specific line (74) but we should be using the above types instead of `Record<string, unknown>`.